app/views/application/_tile_template.html.erb

```diff
@@ -0,0 +1,11 @@
+<div onclick="location.href='<%= destination_path%>';" class="tile">
```

Comment:
This won't work if Javascript is disabled in the Browser. Could use an `<a>` Tag with absolute positioning to achieve the same result.

Reply:
I think many features are dependent of JS already ;). Nonetheless I changed it,  good point. Could you accept?